features_logic.tex

```diff
@@ -0,0 +1,75 @@
+
+SymPy supports construction and manipulation of boolean expressions
+through the \texttt{logic} module. SymPy symbols can be used as 
+propositional variables and also be substituted as \texttt{True} 
+or \texttt{False}. A good number of manipulation features for boolean 
+expressions have been implemented in the \texttt{logic} module.
+
+\subsubsection{Constructing boolean expressions}
+
+A boolean variable can be declared as a SymPy symbol. Python
+operators \&, \textbar  and \textasciitilde are overloaded for logical \texttt{And}, 
+\texttt{Or} and \texttt{negate}. Several others like \texttt{Xor},
+\texttt{Implies} can be constructed with \^{}, \textgreater\textgreater respectively.  
+The above are just a shorthand, expressions can also be constructed
+by directly calling \texttt{And()}, \texttt{Or()}, \texttt{Not()},
+\texttt{Xor()}, \texttt{Nand()}, \texttt{Nor()}, etc.
+The boolean symbols can also be substituted \texttt{True} or \texttt{False}
+
+\begin{verbatim}
+>>> e = (x & y) | z
```

Comment:
IMO, you should import from sympy and create the necessary symbols and stuff. That way somebody following along can just copy and paste the examples in say an IPython shell.